versions.lock

```diff
@@ -114,12 +114,14 @@ io.netty:netty-codec-http:4.1.96.Final (3 constraints: 582436ee)
 io.netty:netty-codec-http2:4.1.96.Final (1 constraints: ed0a94cb)
 io.netty:netty-codec-socks:4.1.96.Final (1 constraints: 160fb96c)
 io.netty:netty-common:4.1.96.Final (12 constraints: 3cb3817d)
-io.netty:netty-handler:4.1.96.Final (3 constraints: 732b553d)
+io.netty:netty-handler:4.1.96.Final (3 constraints: 722b4f3d)
 io.netty:netty-handler-proxy:4.1.96.Final (1 constraints: ed0a94cb)
 io.netty:netty-resolver:4.1.96.Final (2 constraints: 5c1a8332)
+io.netty:netty-tcnative-boringssl-static:2.0.61.Final (1 constraints: d10fc38e)
+io.netty:netty-tcnative-classes:2.0.61.Final (1 constraints: d113ea5d)
```

Comment:
`tcnative-boringssl` is pulled in due to https://issues.apache.org/jira/browse/ZOOKEEPER-4622 which adds more performant SSL support. Do you think we can safely exclude this dependency from Solr? I could not read from the JIRA whether this is opt-in or applies to the zk client, server or both.

Reply:
I don't think we need to exclude anything. I think that dependency is just a shim? Its a few KB if that - https://mvnrepository.com/artifact/io.netty/netty-tcnative-boringssl-static/2.0.61.Final

Based on reading ZOOKEEPER-4622 it looks like the native libraries have to be setup separately (os and classpath).

Reply:
Yea, no big deal with such light weight. Agree.